charts/karpenter/README.md

```diff
@@ -16,7 +16,7 @@ You can follow the detailed installation instruction in the [documentation](http
 helm upgrade --install --namespace karpenter --create-namespace \
   karpenter oci://public.ecr.aws/karpenter/karpenter \
   --version v0.32.1 \
-  --set serviceAccount.annotations.eks\.amazonaws\.com/role-arn=${KARPENTER_IAM_ROLE_ARN} \
+  --set serviceAccount.annotations."eks\.amazonaws\.com/role-arn"=${KARPENTER_IAM_ROLE_ARN} \
```

Comment:
```suggestion
  --set "serviceAccount.annotations.eks\.amazonaws\.com/role-arn=${KARPENTER_IAM_ROLE_ARN}" \
```

Should we double-quote around the whole thing?

Reply:
@jonathan-innis well, maybe it will work, but that worked for me so.

Reply:
I mean double quote only that particular part.

Reply:
+1 to this recommendation.